ui/css/treeherder.css

```diff
@@ -218,9 +218,29 @@ th-watched-repo {
     transition: width 0.2s;
 }
 
+#platform-job-text-search-field-parent {
```

Comment:
Not related to the work, but I wonder if we might take the opportunity now to shorten all these *'platform-job-text'* ids. I can do it later though if you prefer.

I just wonder if `search-field-parent` and `search-field` is sufficient.